raw-data/GM/gm-afr-inventories.tsv

```diff
@@ -5661,7 +5661,7 @@ InventoryID	LanguageCode	LanguageName	SpecificDialect	Phoneme	Notes	Allophones	A
 1359	shk	Shilluk		b				shk.pdf
 	shk	Shilluk		tç	described as 'like the ch in church, but a little further back'			
```

Comment:
while you're editing this inventory: this `tç` looks like a violation of our conventions (should be `cç` right?)